scripts/variantstore/wdl/extract/gvs_avros_to_vds.py

```diff
@@ -0,0 +1,375 @@
+           intermediate_resume_point=int,
+           skip_final_merge=bool,
+           unphase=bool,
+           )
+
+
+def import_gvs(refs: 'List[List[str]]',
+               vets: 'List[List[str]]',
+               sample_mapping: 'List[str]',
+               site_filtering_data: 'List[str]',
+               vqsr_filtering_data: 'List[str]',
+               vqsr_tranche_data: 'List[str]',
+               final_path: 'str',
+               tmp_dir: 'str',
+               truth_sensitivity_snp_threshold: 'float' = 0.997,
+               truth_sensitivity_indel_threshold: 'float' = 0.990,
+               reference_genome='GRCh38',
+               partitions_per_sample=0.35,
+               intermediate_resume_point=0,
+               skip_final_merge=False,
+               unphase=False,
+               ):
+
+    """Import a collection of Avro files exported from GVS.
+    This function is used to import Avro files exported from BigQuery for
+    the GVS database used for the All of Us project. The resulting data type is
+    a :class:`.VariantDataset`, which is a modern representation of sparse cohort-level
+    data in Hail with reference blocks instead of a dense VCF-like representation.
+    This function accepts inputs where the reference and variant data is broken into
+    sample groups (with identical blocking for reference and variant data). Each sample
+    group table is represented by a group of Avro files. Data must be sorted by the
+    genomic coordinate (location) field both **within and between Avro files**. Order
+    of samples at a genomic coordinate is not required.
+    The ``tmp_dir`` argument should refer to a path in network visible storage
+    (Google Bucket, etc) preferably with a lifecycle policy to delete temporary
+    data after a short duration (e.g. 5 days).
+    **Data transformations**
+      - `refs` -- The `state` field is transformed to an int32 `GQ` field. The reference base
+        is added in from a FASTA file to ensure compatibility with Hail functionality that
+        requires an allele at every locus.
+      - `vets` -- The `GT` field is parsed as a Hail :class:`.tcall` and renamed to `LGT`
+        to denote its indexing by local alleles. The `alt` field is split by commas and recorded
+        from strings to integers that refer to an index (0 being ref) into the array of all
+        alleles discovered at a locus. The `GQ` and `RGQ` fields are converted to 32-bit integers.
+      - `site_filtering_data` -- The site filters are converted from a comma-delimited string
+        to a `set<str>` value in Hail. This set contains all unique filters applied, and is
+        an empty set for loci with no record in the input site filtering table.
+      - `vqsr_filtering_data` -- The VQSR data records information specific to a
+        (locus, alternate allele) pair. This data is read in as a dictionary in the row
+        scope of the resulting variant table, where ys keof the dictionary are an alternate
```

Comment:
?ys keof?

Reply:
This is [in the original](https://github.com/hail-is/hail/pull/12017/files#diff-ca5f5dc1df328033aa52f76078920a655e48f6cdb683333dcec0adb65c137c8cR3118)... guessing this meant to say "keys of" ?